en/test_and_ci/continous_integration.md

```diff
@@ -1,3 +1,212 @@
+| [PX4_misc/Firmware-compile](http://ci.px4.io/job/PX4_misc/job/Firmware-compile/) (On Github shown as **Jenkins Compile All Boards**) | Builds specific list of board targets to check if they all compile. | Pushes & Pull Requests (all Webhook events) | [.ci/Jenkinsfile-compile](https://github.com/PX4/PX4-Autopilot/blob/main/.ci/Jenkinsfile-compile) |
+| [PX4_misc/Firmware_update-world_magnetic_model](http://ci.px4.io/job/PX4_misc/job/Firmware_update-world_magnetic_model/) | Create PR updating world magnetic model data | Manual | In Jenkins |
+| [PX4_misc/Firmware_update_nuttx_kconfigs](http://ci.px4.io/job/PX4_misc/job/Firmware_update_nuttx_kconfigs/) | Create PR updating inconsistent NuttX KConfig files | Manual | In Jenkins |
+| [PX4_misc/Firmware_update_submodules](http://ci.px4.io/job/PX4_misc/job/Firmware_update_submodules/) | Create PRs updating the submodules to latest | Manual | In Jenkins |
+| [PX4/*](http://ci.px4.io/job/PX4/) | Main CI pipelines for PX4 ecosystem | NA | NA |
+| [PX4/PX4-Autopilot](http://ci.px4.io/job/PX4/job/PX4-Autopilot/) (On Github shown as **continuous-integration/jenkins/branch or pr-head**) | Updates the `master` metadata files in the S3 bucket | Push to `main` | [Jenkinsfile](https://github.com/PX4/PX4-Autopilot/blob/main/Jenkinsfile) |
+| [PX4/PX4-user_guide](http://ci.px4.io/job/PX4/job/PX4-user_guide/) | Builds the Html version of the user guide from the markdown format and deploys to the web | Updates in `main` (latest) or `v1.*` (past releases) branches | [PX4-user_guide/Jenkinsfile](https://github.com/PX4/PX4-user_guide/blob/main/Jenkinsfile) |
+
+For the full view of all different pipelines, visit [ci.px4.io/](http://ci.px4.io/).
+
+### Hardware testing Jenkins
+
+A Webhook is configured to [px4-jenkins.dagar.ca](http://px4-jenkins.dagar.ca:8080/) for every event of Pushes (new commits, branches) and Pull Requests in PX4-Autopilot repository.
+
+| Pipeline Name | Description | Trigger | Script |
+|---|---|---|---|
+| [PX4-Autopilot](http://px4-jenkins.dagar.ca:8080/job/PX4-Autopilot/) (On Github shown as **continuous-integration/jenkins/branch**) | Runs hardware test script | Pushes & Pull Requests (all Webhook events) | [.ci/Jenkinsfile-hardware](https://github.com/PX4/PX4-Autopilot/blob/main/.ci/Jenkinsfile-hardware) |
+
+Hardware rack is composed of actual hardware connected via USB, each connected with separate Jenkins nodes currently as following:
+
+- `cubepilot_cubeorange_test`: Cubepilot CubeOrange
+- `cuav_x7pro_test`: CUAV X7 Pro
+- `px4_fmu-v4_test`: FMU v4
+- `px4_fmu-v4pro_test`: FMU v4 Pro
+- `px4_fmu-v5_debug`: FMU v5 with debug flag, prints out debug information
+- `px4_fmu-v5_stackcheck`: FMU v5 compiled with stackcheck, detects stack overflow, etc.
+- `px4_fmu-v5_test`: FMU v4
+- `nxp_fmuk66-v3_test`: FMU k66 v3
+
+Note that `_test` label for the target means ... (TODO)
```

Comment:
@bkueng I actually don't know how "stackcheck", "test", etc labels get incorporated into the build process. What does '_test' postfix do regarding the built target binary?

Reply:
It's just different build variants for selecting a certain sets of modules: the label defines which `.px4board` file is used under the board directory.

Reply:
Ahaa true, I forgot about that! Thanks for the pointer 👍